cpp/splinepy/py/py_spline.hpp

```diff
@@ -468,6 +468,46 @@ class PySpline {
     return sampled;
   }
 
+  /**
+   * @brief Evaluate the Jacobian at certain positions
+   *
+   * @param queries position in the parametric space
+   * @param nthreads number of threads for evaluation
+   * @return py::array_t<double>
+   */
+  py::array_t<double> Jacobian(const py::array_t<double> queries,
```

Comment:
As this is just an efficient helper for `spline.deriavtive(queries.repeat(spline.para_dim, axis=0), np.eye(spline.para_dim).repeat(len(queries), axis=0)`, can you make this a separate module function instead of member function?

Reply:
While you are at it, also `EvaluateBoundaryCenters`. Very convenient, but main use for export

Reply:
One more note, if you implement this in `splines/helpers/properties.hpp`, for example, you can directly fill the buffers in `GetBoundaryOrientation`

Reply:
Done!